projects/OG-Analytics/src/main/java/com/opengamma/analytics/financial/provider/description/interestrate/NormalSTIRFuturesSmileProvider.java

```diff
@@ -20,7 +20,7 @@
  * Implementation of a provider of normal volatility (Bachelier model) smile for options on STIR futures. The volatility is time to expiration/strike/delay dependent.
  * The "delay" is the time between expiration of the option and last trading date of the underlying futures.
  */
-public class NormalSTIRFuturesSmileProvider implements NormalSTIRFuturesSmileProviderInterface {
+public class NormalSTIRFuturesSmileProvider implements NormalSTIRFuturesProviderInterface {
```

Comment:
For name consistency, it would be better to rename this provider as "NormalSTIRFuturesExpStrikeProvider"